### ISSUE TITLE
Some suggestions

### ISSUE THREAD

**Issue by odvigajlo** [Mrbt0907/Weather2-Remastered#72]
1. Red Sprites: maybe they could randomly flash above supercells at night? I think that would be cool idea.
2. Hurricane Eye: hurricanes should definitely have eye in center.
3. Different Lightning Colors: Lightning should have different colours. Like blue, purple. And sometimes red in desert
4. Tornado Pickup Rates: tornadoes should have different pickup rates depending on it strengh

**Comment by GGlitchX** [comment]
Different lightning colors don't make sense much. The other ideas are cool, though. I have already mentioned pickup rates though and Mrbt has confirmed they will maybe add it once he's done with most of the update and simulation mode if they have time/or motivation lol.

**Comment by ProfessorFartsalot** [comment]
> Different lightning colors don't make sense much. The other ideas are cool, though. I have already mentioned pickup rates though and Mrbt has confirmed they will maybe add it once he's done with most of the update and simulation mode if they have time/or motivation lol.

Different lightning color actually makes a lot of sense. Sometimes in more severe storms, lightning can be pink, purple, orange, white, and even blue. My family and I watch storms go through all the time and we can confirm with 100% certainty that multi colored lightning is indeed a phenomenon.

**Comment by odvigajlo** [comment]
> > Different lightning colors don't make sense much. The other ideas are cool, though. I have already mentioned pickup rates though and Mrbt has confirmed they will maybe add it once he's done with most of the update and simulation mode if they have time/or motivation lol.
> 
> Different lightning color actually makes a lot of sense. Sometimes in more severe storms, lightning can be pink, purple, orange, white, and even blue. My family and I watch storms go through all the time and we can confirm with 100% certainty that multi colored lightning is indeed a phenomenon.

actually red, orangy lightning means, that air has quite a lot of dust, smoke

**Comment by ProfessorFartsalot** [comment]
Yeah sometimes we have a lot of dry weather so the soil dries out and I assume dust blows around on the road and the severe storms have wind and updrafts that send the dust flying through the air so the lightning strikes it and lights it up with a beautiful sodium orange color
